lang/po/zh_CN.po

```diff
@@ -181919,12 +181918,12 @@ msgstr "8小时"
 
 #: src/iuse_actor.cpp
 msgid "Until you gain a spell level"
-msgstr "直到你获得法术等级"
+msgstr "直到法术升级"
 
 #: src/iuse_actor.cpp
 #, c-format
 msgid "This item casts %s at level %i."
-msgstr "这件魔道具可以施放%s，等级%i。"
+msgstr "这件魔道具可以施放 %i 级 %s。"
 
```

Comment:
This  one is  problematic since it change argument position which will  cause crash.

Reply:
This should be fixed at Transifex side.